app/controllers/container-type.js

```diff
@@ -31,26 +23,52 @@ export default Controller.extend({
       name: 'Name'
     }];
 
-    this.sortProperties = ['name'];
+    // By default, sort alphabetically
+    this.sorts = [
+      {
+        valuePath: 'name',
+        isAscending: true
+      }
+    ]
   },
 
   /**
    * Inspect an instance in the object inspector.
    * Called whenever an item in the list is clicked.
    *
    * @method inspectInstance
-   * @param {Object} obj
+   * @param {Object} instance
    */
-  inspectInstance: action(function(obj) {
-    if (!get(obj, 'inspectable')) {
+  inspectInstance: action(function(instance) {
+    if (!get(instance, 'inspectable')) {
       return;
     }
     this.port.send('objectInspector:inspectByContainerLookup', {
-      name: get(obj, 'fullName')
+      name: get(instance, 'fullName')
     });
   }),
 
   sendContainerToConsole: action(function() {
     this.port.send('objectInspector:sendContainerToConsole');
   }),
+
+  @action
+  updateSorts(sorts) {
+    let hasExistingSort = this.sorts && this.sorts.length;
```

Comment:
Seems like this is always expected to be true right? Since we initialize it to non-empty and also normalizes it to be non-empty in this setter.

Reply:
👍  eliminating...